internal/generator/vector/output/loki/bearer-token.go

```diff
@@ -0,0 +1,17 @@
+package loki
```

Comment:
underscores in file names is the golang way

Reply:
sure

Reply:
btw, there are many other files i have named with `-` , need to change them all to `_` ?

Reply:
updated